examples/common/pigweed/rpc_console/py/chip_rpc/console.py

```diff
@@ -94,11 +102,26 @@ def _start_ipython_terminal(client: HdlcRpcClient) -> None:
         channel_client=client.client.channel(1),
         rpcs=client.client.channel(1).rpcs,
         protos=client.protos.packages,
+        # Include the active pane logger for creating logs in the repl.
+        LOG=_DEVICE_LOG,
     )
 
-    print(__doc__)  # Print the banner
-    IPython.terminal.embed.InteractiveShellEmbed().mainloop(
-        local_ns=local_variables, module=argparse.Namespace())
+    welcome_message = cleandoc("""
+        Welcome to the CHIP RPC Console!
+
+        Press F1 for help.
+        Example commands:
+
+          rpcs.chip.rpc.DeviceCommon.GetDeviceInfo()
+
+          LOG.warning('Message appears console log window.')
+    """)
+
+    embed(global_vars=local_variables,
+          local_vars=None,
+          loggers=[_DEVICE_LOG],
+          repl_startup_message=welcome_message,
```

Comment:
You could add an `app_title='CHIP Console',` kwarg for some branding.

Reply:
Done, thanks for the tip!